renderlib/IRenderWindow.h

```diff
@@ -18,8 +20,10 @@ class IRenderWindow
   virtual void resize(uint32_t w, uint32_t h, float devicePixelRatio = 1.0f) = 0;
   virtual void cleanUpResources() {}
 
-  // an interface for reporting statistics and other data updates
-  virtual CStatus* getStatusInterface() { return nullptr; }
+  // An interface for reporting statistics and other data updates
+  // The IRenderWindow is the first to create this object but it can get shared with other widgets in the gui.
+  // Alternative impl could be to create at app layer and pass down into renderers

```

Comment:
This is a good comment; a nit might be to explain why you used one solution over the alternate.